server/configHelpers.js

```diff
@@ -162,7 +166,42 @@ async function testConfigOptions(options) {
     return [true, clientErrors];
   }
 
-  return [false];
+  return [false, null];
+}
+
+/*
+ * Simple utility for getting a default config with logging
+ * for certain edge cases
+ * @param {Config} bpanelConfig
+ * @returns {Config}
+ */
+
+function getDefaultConfig(bpanelConfig) {
+  assert(
+    bpanelConfig instanceof Config,
+    'Need the main bcfg for the app to get default configs'
+  );
+  const clientConfigs = loadClientConfigs(bpanelConfig);
+  let defaultClientConfig = clientConfigs.find(
```

Comment:
Could there ever be more than 1 default config?

Reply:
yes and no. It's partly dictated by file name. Even if there are more than one though, that's kind of something for the user to update. This is just so that there's a fallback when you first start up bPanel. So if I'm the user and I see there are two clients with the `id` of `default`, I'd end up going in to the settings myself and changing the name to clarify. It might be worth it to update this API at some point anyway once the clients UI flow is a little more settled (e.g. I want to test these setups from scratch, and also when there are no configs at all).